scene/gui/rich_text_label.cpp

```diff
@@ -359,9 +360,12 @@ int RichTextLabel::_process_line(ItemFrame *p_frame, const Vector2 &p_ofs, int &
 						strikethrough = true;
 					}
 
-					fade = _fetch_by_type<ItemFade>(text, ITEM_FADE);
-					_fetch_item_stack<ItemFX>(text, fx_stack);
+					ItemText *text = static_cast<ItemText *>(it);
 
+					Item *fade_item = it;
+					if (_find_by_type(fade_item, ITEM_FADE)) {
```

Comment:
The fade issue is likely a regression from the _find_by_type function not returning ITEM_FADE types properly?

Reply:
Yeah this will be the issue.

Reply:
Fixed now.